src/commands/workspace/logs.ts

```diff
@@ -47,21 +48,23 @@ export default class Logs extends Command {
     const cheTasks = new CheTasks(flags)
     const k8sTasks = new K8sTasks()
 
+    const follow = flags.follow
     const tasks = new Listr([], { renderer: flags['listr-renderer'] as any })
     tasks.add(k8sTasks.testApiTasks(flags, this))
     tasks.add(cheTasks.verifyCheNamespaceExistsTask(flags, this))
-    if (!flags.follow) {
+    if (!follow) {
       tasks.add(cheTasks.verifyWorkspaceRunTask(flags, this))
     }
     tasks.add(cheTasks.workspaceLogsTasks(flags))
+    tasks.add(cheTasks.namespaceEventsTask(flags, this, follow))
```

Comment:
Well, it will listen to events in a namespace where Che Server is deployed and it's not really thing we should do...
Maybe instead it would be better to collects events in namespace where workspace is really started (default value now is `<username>-che`). You can fetch this info from Che Server/api/workspace/{workspaceid}#attributes.infrastructureNamespace.

Please also consider filter objects by workspaceId label.